master/buildbot/www/oauth2.py

```diff
@@ -226,12 +226,14 @@ def __init__(self,
                  **kwargs):
 
         super().__init__(clientId, clientSecret, autologin, **kwargs)
+        self.apiResourceEndpoint = None
         if serverURL is not None:
             # setup for enterprise github
-            if serverURL.endswith("/"):
-                serverURL = serverURL[:-1]
+            serverURL = serverURL.rstrip("/")
```

Comment:
Also another question. The if here before removed the '/' at the end of the string.
Shouldn't it be an rstrip that does the same?

Reply:
Agreed, this looks better.